webpack-dev-server.el

```diff
@@ -62,18 +60,11 @@
   (kill-webpack-dev-server)
   (setq-local default-directory webpack-dev-server-project-root)
   (let ((webpack-dev-server-buf (get-buffer-create webpack-dev-server-buf-name)))
-    (display-buffer
-     webpack-dev-server-buf
-     '((display-buffer-at-bottom
-        display-buffer-pop-up-window
-        display-buffer-reuse-window)
-       (window-height . 30)))
-    (select-window (get-buffer-window webpack-dev-server-buf))
+    (switch-to-buffer-other-window webpack-dev-server-buf)
     (make-term "webpack-dev-server" "/bin/bash")
     (term-mode)
     (term-char-mode)
     (term-set-escape-char ?\C-x)
-    (setq-local term-buffer-maximum-size webpack-dev-server-height)
```

Comment:
I think the biggest problem with this is that it no longer "pushes" out old error messages. So, if I'm making changes to my javascript and there's an error and I do `next-error`, I won't go to the top of the _visible_ buffer I'll go to the next error I haven't visited yet which could be from the last cycle. I'll test it.